### DISCUSSION TITLE
I'm a newbie

### DISCUSSION THREAD

**Discussion by syamadiatsyah** [ProjectOpenSea/opensea-js#1293] [Q&A]
I want to ask, I want to mint my artwork on OpenSea for the first time, is there any cost for initially releasing our work as an NFT? I still don't understand the features on OpenSea.

**Comment by ryanio** [comment]
Yes you can browse to opensea.io/studio . There is a cost to deploy the contract first and mint each NFT, but if you use a L2 network like Optimism or Arbitrum it is much cheaper than mainnet.
